test/com/googlecode/jmxtrans/model/output/CloudWatchWriterTests.java

```diff
@@ -0,0 +1,21 @@
+package com.googlecode.jmxtrans.model.output;
+
+import com.googlecode.jmxtrans.model.Query;
+import com.googlecode.jmxtrans.util.ValidationException;
+import org.junit.Test;
+
+/**
+ * Tests for {@link com.googlecode.jmxtrans.model.output.CloudWatchWriter}.
+ *
+ * @author Sascha Moellering <moelleri@amazon.de>
+ */
+public class CloudWatchWriterTests {
```

Comment:
Testing with an AWS account would be integration testing. I care much more about unit testing. You have quite a bit of logic that could be tested without accessing AWS. For example, the `convertToDouble()` and most of the `getRegion()` methods can be tested without accessing AWS.